assets/js/components/Programmes.vue

```diff
@@ -1,12 +1,9 @@
 <template>
-  <div class="sidebar-tab-pane programmes"
-       v-bind:class="{
-        'is-selected': selected,
-        'is-loading': loading
-       }">
-    <ul class="sidebar-tab-result-list" v-if="hasData">
+  <section class="sidebar-section-wrapper programmes"
+             role="tabpanel">
```

Comment:
`class="sidebar-section-wrapper"` and `role="tabpanel"` don't belong here, they should be set by the parent